src/pages/HomePage/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { CategoryCard } from '../CategoryCard';
+import { getProducts } from '../../../../helpers/getProducts';
+import { ProductCategories } from '../../../../types/ProductCategories';
+import { Product } from '../../../../types/ProductCard';
+import './Categories.scss';
+
+export const Categories: React.FC = () => {
+  const [phoneModelsAmount, setPhoneModelsAmount] = useState<number>(0);
+  const [tabletModelsAmount, setTabletModelsAmount] = useState<number>(0);
+  const [accessoryModelsAmount, setAccessoryModelsAmount] = useState<number>(0);
+
+  useEffect(() => {
+    getProducts(ProductCategories.Phones).then((phones: Product[]) => {
```

Comment:
Good job! 
Just as a sneak-peak in future - usually, backend should return you data in format: 

```
{
  data: [],
  count: 0
}
```

Frontend should never take care for such things, like counting products on it's own( how can you implement a pagination in this case?).